config/application.yml

```diff
@@ -4,3 +4,5 @@ elasticsearch_server: <%= ENV['SEARCH_SERVER'] || 'http://localhost:9200' %>
 max_deep_search_comment_count: 5000
 default_locale: <%= ENV['SERVICE_LANGUAGE'] || 'en-US' %>
 manual_pagination_batch_size: <%= ENV['MANUAL_PAGINATION_BATCH_SIZE'] || 500 %>
+thread_response_default_size: 100
```

Comment:
https://github.com/edx/edx-platform/blob/master/common/static/common/js/discussion/views/discussion_thread_view.js#L322

The first load of a single thread is 25 responses, then the button at the bottom to load more will load another 100.